store/src/java/com/zimbra/cs/imap/ImapPath.java

```diff
@@ -58,8 +58,7 @@ enum Scope { UNPARSED, NAME, CONTENT, REFERENCE };
             FOLDER_ENCODING_CHARSET = Charset.forName("imap-utf-7");
         } catch (Exception e) {
             ZimbraLog.imap.error(
-                "could not load imap-utf-7 charset (perhaps zimbra-charset.jar is not in the jetty endorsed directory)",
-                e);
+                "could not load imap-utf-7 charset (perhaps zimbra-charset.jar is not in the jetty endorsed directory)");
```

Comment:
why is the exception obj logging removed?

Reply:
It was injecting noise into the test output with a stack trace that adds nothing. I can put it back. It just doesn't seem necessary with a message that descriptive.